docs/user-guide/at-tls-configuration.md

```diff
@@ -1,23 +1,642 @@
+
+```bash
+TTLSRule ApimlServiceClientRule
+{
+  LocalAddr All
+  LocalPortRange 1024-65535
+  RemoteAddr All
+  RemotePortRange 8080 # Set to range of ports where services are listening
+  Jobname ZWE1AG* # Generate according to zowe.job.prefix in zowe.yaml
+  Direction Outbound
+  TTLSGroupActionRef ClientGroupAction
+  TTLSEnvironmentActionRef ApimlClientEnvironmentAction
+  TTLSConnectionActionRef ApimlNoX509ClientConnAction # Do not send X.509 Client Certificates
+}
+
+TTLSConnectionAction ApimlNoX509ClientConnAction
+{
+  HandshakeRole Client
+  TTLSCipherParmsRef CipherParms
+  TTLSConnectionAdvancedParmsRef ApimlClientNoX509ConnAdvParms
+}
+
+TTLSConnectionAdvancedParms ApimlClientNoX509ConnAdvParms
+{
+  ApplicationControlled Off
+  CertificateLabel # Leave empty to avoid sending a client certificate (i.e. if the keyring has a default certificate)
+  SecondaryMap Off
+}
+```
+
+:::important
+
+- Outbound connections from the Gateway to southbound services (onboarded services) must not send the server certificate if the service accepts X.509 Client Certificate authentication. If the server certificate is sent, it is the server user who would be authenticated.
+
+:::
+
+### Ciphers
+
+:::note
+This list of ciphers is provided as an example only. Actual ciphers should be customized according to your specific configuration.
+:::
+
+The list of supported ciphers should be constructed according to the TLS supported versions.
+Ensure that the cipher list has matches with non-AT-TLS-aware clients.
+
+<details>
+<summary>Click here for an example of Cipher parameters.</summary>
+
+```bash
+TTLSCipherParms CipherParms
+{
+  V2CipherSuites TLS_RC4_128_WITH_MD5
+  V2CipherSuites TLS_RC4_128_EXPORT40_WITH_MD5
+  V2CipherSuites TLS_RC2_CBC_128_CBC_WITH_MD5
+  V2CipherSuites TLS_RC2_CBC_128_CBC_EXPORT40_WITH_MD5
+  V2CipherSuites TLS_RC2_CBC_128_CBC_EXPORT40_WITH_MD5
+  V2CipherSuites TLS_DES_192_EDE3_CBC_WITH_MD5
+  V3CipherSuites TLS_ECDHE_ECDSA_WITH_AES_128_CBC_SHA256
+  V3CipherSuites TLS_ECDHE_ECDSA_WITH_AES_256_CBC_SHA384
+  V3CipherSuites TLS_ECDHE_ECDSA_WITH_AES_128_GCM_SHA256
+  V3CipherSuites TLS_ECDHE_ECDSA_WITH_AES_256_GCM_SHA384
+  V3CipherSuites TLS_ECDHE_RSA_WITH_AES_256_GCM_SHA384
+  V3CipherSuites TLS_AES_128_GCM_SHA256
+  V3CipherSuites TLS_AES_256_GCM_SHA384
+  V3CipherSuites TLS_CHACHA20_POLY1305_SHA256
+}
+```
+
+</details>
+
+## Using AT-TLS for API ML in High Availability
+
+AT-TLS settings for a Zowe installation configured in High Availability mode do not differ extensively. Changes need to be made to the previously described rules to allow for cross-lpar communication:
+
+Ensure that the `RemoteAddr` setting in the rules accounts for the following connections:
+
+- Discovery Service to Discovery Service. This is the replica request.
+- Gateway Service to southbound services (including app-server and ZSS) running in another LPAR.
+- Southbound services to Discovery Service. This applies during onboarding.
```

Comment:
```suggestion
- Southbound services to Discovery Service. This applies during onboarding.
- Southbound services to Gateway Service if it is onboarded with authentication scheme `zoweJwt` or uses OIDC. This applies during credential verification.
```

Reply:
This is unrelated to High Availability, I'll try to find a place to mention it.